connector/nopconnector/README.md

```diff
@@ -0,0 +1,21 @@
+# Nop Connector
+
+The `nopconnector` can be used to chain pipelines of the same type together.
+For example, it can replicate a signal to multiple pipelines so that each pipeline
+can process the signal independently in varying ways. Alternately, it can be used to
+merge two pipelines together so they can be processed as one pipeline.
```

Comment:
A paragraph here with a couple of use-cases would be nice to have. I myself wouldn't know why I would need this.

Reply:
Added